test/unit/test_cache.py

```diff
@@ -281,47 +282,47 @@ def setUp(self):
         # dropping bar.table2 should only drop itself
         self.cache.drop(make_relation('bar', 'table2'))
-        self.assertEqual(len(self.cache.get_relations('foo')), 3)
-        self.assertEqual(len(self.cache.get_relations('bar')), 1)
+        self.assertEqual(len(self.cache.get_relations('dbt', 'foo')), 3)
+        self.assertEqual(len(self.cache.get_relations('dbt', 'bar')), 1)
         self.assertEqual(len(self.cache.relations), 4)
 
     def test_drop_many(self):
         # dropping foo.table1 should drop everything but bar.table2.
         self.cache.drop(make_relation('foo', 'table1'))
-        self.assertEqual(len(self.cache.get_relations('foo')), 0)
-        self.assertEqual(len(self.cache.get_relations('bar')), 1)
+        self.assertEqual(len(self.cache.get_relations('dbt', 'foo')), 0)
+        self.assertEqual(len(self.cache.get_relations('dbt', 'bar')), 1)
         self.assertEqual(len(self.cache.relations), 1)
 
     def test_rename_root(self):
         self.cache.rename(make_relation('foo', 'table1'),
                           make_relation('bar', 'table1'))
-        retrieved = self.cache.relations[('bar', 'table1')].inner
+        retrieved = self.cache.relations[('dbt', 'bar', 'table1')].inner
         self.assertEqual(retrieved.schema, 'bar')
         self.assertEqual(retrieved.identifier, 'table1')
-        self.assertEqual(len(self.cache.get_relations('foo')), 2)
-        self.assertEqual(len(self.cache.get_relations('bar')), 3)
+        self.assertEqual(len(self.cache.get_relations('dbt', 'foo')), 2)
+        self.assertEqual(len(self.cache.get_relations('dbt', 'bar')), 3)
 
         # make sure drops still cascade from the renamed table
         self.cache.drop(make_relation('bar', 'table1'))
-        self.assertEqual(len(self.cache.get_relations('foo')), 0)
-        self.assertEqual(len(self.cache.get_relations('bar')), 1)
+        self.assertEqual(len(self.cache.get_relations('dbt', 'foo')), 0)
+        self.assertEqual(len(self.cache.get_relations('dbt', 'bar')), 1)
         self.assertEqual(len(self.cache.relations), 1)
 
     def test_rename_branch(self):
         self.cache.rename(make_relation('foo', 'table3'),
                           make_relation('foo', 'table2'))
-        self.assertEqual(len(self.cache.get_relations('foo')), 3)
-        self.assertEqual(len(self.cache.get_relations('bar')), 2)
+        self.assertEqual(len(self.cache.get_relations('dbt', 'foo')), 3)
+        self.assertEqual(len(self.cache.get_relations('dbt', 'bar')), 2)
 
         # make sure drops still cascade through the renamed table
         self.cache.drop(make_relation('foo', 'table1'))
-        self.assertEqual(len(self.cache.get_relations('foo')), 0)
-        self.assertEqual(len(self.cache.get_relations('bar')), 1)
+        self.assertEqual(len(self.cache.get_relations('dbt', 'foo')), 0)
+        self.assertEqual(len(self.cache.get_relations('dbt', 'bar')), 1)
         self.assertEqual(len(self.cache.relations), 1)
```

Comment:
can you add multi database testing here? e.g. two schemas named `foo` with different database names